docs-src/python/choose-dev-cluster.md

```diff
@@ -0,0 +1,133 @@
+---
+id: choose-dev-cluster
+title: Choose a development Cluster
+sidebar_label: Choose dev Cluster
+description: Discover which development Cluster you should choose
+tags:
+  - developer guide
+  - python sdk
+---
+
+**Which development Cluster should you choose?**
+
+We recommend choosing a development environment based on your requirements.
+
+The source code for the Temporal Server (the orchestrating component of the Temporal Cluster) is licensed under the MIT open source license. So, in theory, anyone can take the Temporal Server code and run their Temporal Platform in any number of creative ways.
+
+However, for most developers we recommend starting by choosing one of the following:
+
+- [Local development server](#local-dev-server)
+- [Temporal Cloud](#temporal-cloud)
+- [Self-hosted Temporal Cluster](#self-hosted-temporal-cluster)
+
+:::info Temporal does not directly run your code
+
+Keep in mind that in every scenario, the “Temporal Platform” does not host and run your Workers (application code).
+It is up to you, the developer, to host your application code.
+The Temporal Platform ensures that properly written code durably executes in the face of platform-level failures.
+
+:::
+
+### Local dev server
+
+**When to use a local development server?**
+
+We recommend using the local development server if you are new to Temporal, or want to start something from scratch and don’t have a self-hosted environment ready or want to pay for a Temporal Cloud account.
+
+The Temporal CLI comes bundled with a development server and provides a fast way to start running Temporal Applications.
+
+However, the local development server does not emit any metrics.
+If you are eager to to set up Cluster-level metrics for performance tuning, we recommend using a self-hosted Cluster or Temporal Cloud.
+
+#### Start the dev server
+
+**How to start a local development server**
+
+If you have successfully installed the Temporal CLI, open a new terminal and run the following command:
+
+```bash
+temporal server start-dev
```

Comment:
Would we consider mentioning the support of persistent DB via https://docs.temporal.io/cli/cmd-options#db-filename as an option?

Users in the community do ask questions about how to keep the workflows around in the dev env.

Reply:
```bash
temporal server start-dev --db-filename temporal.db
```

This command automatically starts the Temporal Web UI, creates a default Namespace, and creates a persistence database.

Reply:
I'm not sure in Python setup is the place to document the different options of the CLI's start-dev (even if this is a common one)